src/MembershipsSDKAdapter.js

```diff
@@ -117,11 +136,17 @@ export default class MembershipsSDKAdapter extends MembershipsAdapter {
       inMeeting: null,
     });
 
+    const currentUser$ = new PeopleSDKAdapter(this.datasource).getMe();
+
     const members$ = defer(() => this.datasource.memberships.list({
       roomId: roomID,
       max: MAX_MEMBERSHIPS,
-    })).pipe(
-      map((page) => page.items.map(membershipToMember)),
+    }));
+
+    const orderedMembers$ = currentUser$.pipe(
```

Comment:
try using [combineLatest](https://rxjs-dev.firebaseapp.com/api/index/function/combineLatest)

Reply:
Apparently combineLatest is deprecated and it recommends me to use pipe to map instead
![deprecated_combineLatest](https://user-images.githubusercontent.com/56340398/114694021-d6047f80-9d22-11eb-961a-825007c86c0e.png)

Reply:
I think the `resultSelector` parameter is deprecated, not the operator itself. Maybe you called the operator like this `combineLatest($a, $b)` instead of `combineLatest([$a, $b])`?

Reply:
Yes, i just tried calling the operator like this combineLatest([$a, $b]), but it's still deprecated
![deprecated_combine](https://user-images.githubusercontent.com/56340398/114834223-0575c380-9dd9-11eb-8ddc-51056f682c08.png)